### ISSUE TITLE
"out var _" crashes the compiler

### ISSUE THREAD

**Issue by paveliak** [dotnet/roslyn#22438]
**Version Used**:
Microsoft Visual Studio Enterprise 2017
Version 15.0.26228.9 D15RTWSVC
Microsoft .NET Framework
Version 4.7.02046

**Steps to Reproduce**:
1. Create C# project
2. Add [Moq](https://www.nuget.org/packages/Moq) package
3. Compile this program and observe "error MSB6006: "csc.exe" exited with code -2146232797."

```
namespace CSharpCompilerCrash
{
    class Program
    {
        static void Main(string[] args)
        {
            var foo = new Moq.Mock<Foo>();
            foo.Setup(x => x.F(out var _));
        }
    }

    abstract class Foo
    {
        public abstract void F(out int bar);
    }
}
```
**Expected Behavior**: Program successfully compiles

**Actual Behavior**: C# compiler crashes

**Comment by paveliak** [comment]
A colleague of mine pointed out that I should try updating to the latest version and indeed this issue is fixed in VStudio version 15.3.5. Sorry for the noise. Feel free to close.

**Comment by jcouv** [comment]
Yes, I think this was fixed already. I'll go ahead and close.
Thanks
